userguide/plugins.rst

```diff
@@ -159,6 +159,11 @@ When a newer version of a plugin becomes available in the official
 repository, update the plugin jail by navigating to the
 :menuselection:`Jails` page and clicking |ui-options| and
 :guilabel:`Update`.
+
+.. note:: Plugins are updated using :samp:`iocage update {UUID} | {NAME}`
+   and upgraded using :samp:`iocage upgrade {UUID} | {NAME} -r {RELEASE}`.
+   Manually created jails use :samp:`pkg update` and :samp:`pkg upgrade`.
```

Comment:
They use the same iocage interfaces as plugins. iocage adapts in the background. So this line can be removed.